ludwig/train.py

```diff
@@ -157,6 +157,7 @@ def train_cli(
     if HYPEROPT in config:
         if not query_yes_no(HYPEROPT_WARNING + CONTINUE_PROMPT):
             exit(1)
+        config.pop(HYPEROPT)
```

Comment:
This looks fine to me... but why does the presence of hyperopt section cause issues?  Add a comment to explain or `#TODO` here to investigate

Reply:
Addressed! Going to merge. Linked an issue and added a Todo.